spec/fixtures/yes_no_dont_know_question.html

```diff
@@ -0,0 +1,28 @@
+<form action="/pension-type" accept-charset="UTF-8" method="post">
+  <input name="question_id" type="hidden" value="test_question" />
+
+  <h2 id="test-question">Test question</h2>
+
+
```

Comment:
I went with a  different approach to testing for that very reason. I will kill this file.